### DISCUSSION TITLE
url 작성 방법

### DISCUSSION THREAD

**Discussion by Go-EunLee** [yeojeong-0824/TeamBack#2] [General]
url은 어떻게 작성할까요?? 저는 board/[기능]/[boardId]  이런식으로 작성해요

**Comment by mini0192** [comment]
저는 board/[권한]/[기능] 이런 식으로 작성해요

**Reply by mini0192** [reply]
이 부분은 프론트랑 얘기도 해봐야 할 것 같아요!

**Reply by Go-EunLee** [reply]
그럼 일반 회원이면 권한은 board/general/write 이런식으로 작성하나요?

**Reply by mini0192** [reply]
모든 사용자
/board/write

로그인 사용자
/board/authed/write

관리자
/board/admin/write

저는 이런식으로 작성해요!

**Comment by ssbin0916** [comment]
전 board 리퀘스트 매핑하고 나머지는 기능/{id}

**Comment by mini0192** [comment]
이 부분은 내일 회의 때 결정하는 건 어떨까요? 프론트와 얘기를 해봐야 할 것 같아요
